src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/AdoNet/SqlClient/SqlClientConstants.cs

```diff
@@ -0,0 +1,28 @@
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet.SqlClient
+{
+    internal static class SqlClientConstants
+    {
+        public const string SqlCommandIntegrationName = "SqlCommand";
+
+        public static class SystemData
+        {
+            public const string AssemblyName = "System.Data";
+            public const string SqlClientAssemblyName = "System.Data.SqlClient";
```

Comment:
nit: Can we rename these fields so it doesn't look like `System.Data.dll` is the default assembly and `System.Data.SqlClient.dll` is a special case? Maybe something like `SystemDataAssemblyName` and `SystemDataSqlClientAssemblyName`?

Reply:
I understand what you mean but I came with those names because the container class is called `SystemData` so both calls are: `SqlClientConstants.SystemData.AssemblyName` and `SqlClientConstants.SystemData.SqlClientAssemblyName`

Reply:
When you write it out like that it makes more sense!

Although, now I kinda want them all to be `SqlClientConstants.<Assembly>.AssemblyName` 😅 
```
SqlClientConstants.SystemData.AssemblyName 
SqlClientConstants.SystemDataSqlClient.AssemblyName
SqlClientConstants.MicrosoftDataSqlClient.AssemblyName
```

No need to change it. Level of care: low

Reply:
Done in 9b0ba37 😉